src/MetricScene/RelatedLogs/RelatedLogsOrchestrator.ts

```diff
@@ -152,13 +156,9 @@ export class RelatedLogsOrchestrator {
     datasources.forEach((datasource) => {
       const queryRunner = new SceneQueryRunner({
         datasource: { uid: datasource.uid },
-        queries: [],
-        key: `related_logs_check_${datasource.uid}`,
-      });
-
-      // Build and set queries
-      queryRunner.setState({
         queries: this.getLokiQueries(datasource.uid),
+        key: `related_logs_check_${datasource.uid}`,
+        $timeRange: this._metricScene.state.$timeRange,
```

Comment:
We weren't passing this before, but AFAIK we haven't encountered bugs related to the time ranges of related logs